docs/source/specs/openapi.json

```diff
@@ -3443,6 +3460,7 @@
                             "source_type",
                             "authentication",
                             "billing_source",
+                            "koku_uuid",
```

Comment:
whoa, how did this sneak back in?  Please remove.  This is what was breaking the new Sources API IQE integration yesterday.